### ISSUE TITLE
Accessibility Violations in v6.1.11

### ISSUE THREAD

**Issue by danderson421** [fullcalendar/fullcalendar#7707]
### Checklist

- [X] I've already searched through [existing tickets](https://github.com/fullcalendar/fullcalendar/issues)
- [X] Other people will find this feature useful

### Is this feature for a specific connector (React/Angular/etc)?

No connector in particular

### Feature Description

There are many accessibility "Rule" violations in month view. Using the new browser plugin: IBM Equal Access Accessibility Checker, the results indicate 65 Violations in month view.

Violations 60 - Table structure elements cannot specify an explicit 'role' within table containers
Violation 2 - Browsers ignore the explicit and implicit ARIA roles of the descendants of certain elements
Violation 2 - For a complex data table, all <th> and <td> elements must be related via 'header' or 'scope' attributes
Violation 1- Interactive component must have a programmatically associated name

[https://www.ibm.com/able/toolkit/tools](url)

**Comment by acerix** [comment]
I believe these existing tickets capture these issues:
https://github.com/fullcalendar/fullcalendar/issues/7568
https://github.com/fullcalendar/fullcalendar/issues/7567
https://github.com/fullcalendar/fullcalendar/issues/6641
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.
